Library/src/Systems/Assets/AssetSystem.cpp

```diff
@@ -37,6 +37,9 @@ namespace chs = services::generated::prototypeservice;
 constexpr int DEFAULT_SKIP_NUMBER		= 0;
 constexpr int DEFAULT_RESULT_MAX_NUMBER = 100;
 
+static const char* MATERIAL_ASSET_COLLECTION_NAME_PREFIX = "ASSET_COLLECTION_MATERIAL_";
+static const char* MATERIAL_ASSET_NAME_PREFIX			 = "ASSET_MATERIAL_";
+static const char* MATERIAL_FILE_NAME_PREFIX			 = "ASSET_MATERIAL_FILE_";
```

Comment:
[Consider] Why static? Moving these into the anonymous namespace below seems a better option. (I'm assuming you're worried about linkage?)

Reply:
Yep, unamed namespace is better.